feed/tests/test_feed_views.py

```diff
@@ -45,3 +52,60 @@ def test_detail_view_page_for_invalid_post_id(self, client):
         max_post_id = Post.posts.all().count()
         response = client.get(POST_DETAIL_URL + str(max_post_id + 1) + '/')
         assert response.status_code == PAGE_NOT_FOUND
+
+
+@pytest.fixture
+def liked_post(post, user):
+    post.likes.add(user)
+    return post
+
+
+class TestLikeView:
+    def test_like_view_return_warning_when_logged_out(self, post, client):
+        # Testing that when logged out, if trying to access the like url
+        # a warning message returns
+        post_to_check_likes = Post.posts.filter(title=post.title, content=post.content, author=post.author).first()
+        response = client.get(post_like_url(post_to_check_likes.id))
```

Comment:
I would advisde. to check that if user is not logged it, he is forwarded to login page.

Reply:
added